packages/ui/src/ui/components/ColumnCell/ColumnCell.tsx

```diff
@@ -0,0 +1,170 @@
+import React, {useState} from 'react';
+import cn from 'bem-cn-lite';
+import copyToClipboard from 'copy-to-clipboard';
+
+import {Button, Flex, Icon as UIKitIcon} from '@gravity-ui/uikit';
+import {Eye} from '@gravity-ui/icons';
+
+import unipika from '../../common/thor/unipika';
+import ClipboardButton from '../../components/ClipboardButton/ClipboardButton';
+import YqlValue from '../../components/YqlValue/YqlValue';
+import Yson from '../../components/Yson/Yson';
+import Label from '../../components/Label/Label';
+import {TypeArray} from '../../components/SchemaDataType/dataTypes';
+import {Tooltip} from '../../components/Tooltip/Tooltip';
+import {UnipikaSettings} from '../../components/Yson/StructuredYson/StructuredYsonTypes';
+
+import './ColumnCell.scss';
+
+const block = cn('yt-column-cell');
+
+function unquote(v: string) {
```

Comment:
I would put these microfunctions in helpers and put them next to the component. It's a little cleaner for me. It's not a mistake, so see for yourself.